docker-compose.yml

```diff
@@ -15,3 +15,10 @@ services:
     restart: unless-stopped
     environment:
       - PYTHONUNBUFFERED=1
+  prometheus:
```

Comment:
no need for prometheus container, one.sce can get metrics from the fastapi server